src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/io/UrlSource.kt

```diff
@@ -0,0 +1,23 @@
+package com.cognifide.gradle.aem.tooling.tail.io
+
+import com.cognifide.gradle.aem.instance.Instance
+import com.cognifide.gradle.aem.tooling.tail.LogSource
+import com.cognifide.gradle.aem.tooling.tasks.Tail
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+class UrlSource(private val aemInstance: Instance) : LogSource {
+
+    override fun <T> readChunk(parser: (BufferedReader) -> T) = aemInstance.sync {
+        get(ERROR_LOG_ENDPOINT) {
+            BufferedReader(InputStreamReader(asStream(it))).use(parser)
+        }
+    }
+
+    companion object {
+        const val ERROR_LOG_ENDPOINT = "/system/console/slinglog/tailer.txt" +
+                "?_dc=1520834477194" +
```

Comment:
`dc` is probably current timestamp, investigate that param value being passed in CRX DE Lite and reflect that value generation

Reply:
This might change (simplify) the solution, I would leave it for the next step since it's working.